src/main/java/Main.java

```diff
@@ -0,0 +1,81 @@
+import model.Order;
+import model.Product;
+import repo.OrderRepo;
+import repo.ProductRepo;
+import service.ShopService;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        // main menu
+        mainmenu();
```

Comment:
```suggestion
        printMainMenu();
```

Typischerweise hat man bei Methoden immer ein Verb das sagt was die Methode macht.
Zweck: bessere Lesbarkeit und mehr Verständnis (sonst frage ich mich "was ist mit dem main menu?/ was soll damit passieren?")